easybuild/easyconfigs/b/BWA/BWA-0.7.13-foss-2015b.eb

```diff
@@ -0,0 +1,32 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Cyprus Institute / CaSToRC, Uni.Lu/LCSB, NTUA
+# Authors::   George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+# 
+# Version >= 0.7.13
+# Author: Adam Huffman
+# The Francis Crick Institute 
+##
+
+name = 'BWA'
+version = '0.7.13'
+
+homepage = 'https://github.com/lh3/bwa'
```

Comment:
@verdurin : coming late to the party, but just to mention that if you have a hash check, it can only be an advantage to have two download sources (ie. it's beneficial for the next user of this...)